src/awesome_o/slack.clj

```diff
@@ -53,10 +62,5 @@
     (select-next-slackmaster)
     (doseq [person (state/persons-born-today)]
       (say (format "Today is @%s's birthday! Happy birthday!" person)))
-    (when (time/monday-today?)
-      (let [devs (shuffle (state/available-devs))
-            gbgs (shuffle (state/get-available-people-in-location "göteborg"))
-            meeting-master (pingify [(rand-nth gbgs)])
-            honeybadgers (->> devs (take 2) pingify)]
-        (say (str "Honeydager monday! ping: " honeybadgers) :channel "dev")
-        (say (str "Todays meeting master for dev this week is " meeting-master))))))
+    (when (time/monday-today?) monday-announcements)
```

Comment:
OK I guess almost caricatural first PR review comment: you forgot a pair of parenthesis around "monday-announcements"